lib/launchers/dns_utils.rb

```diff
@@ -0,0 +1,71 @@
+#Define two functions to creat/delete dns records, used for osp and vsphere platform
+def create_dns_records(dns_solution, instances_name_prefix, ip_api, ip_api_int, ip_apps)
+  base_domain = nil
+  logger.info("DNS_SOLUTION: #{dns_solution}, PREFIX: #{instances_name_prefix}, IPI_API: #{ip_api}, IP_API_INT: #{ip_api_int}, IP_APPS: #{ip_apps}")
+  case dns_solution
+  when "dyndns"
```

Comment:
I do not think we should consider dyndns any more.

Reply:
yeah, we can take that out